src/components/EventForm.tsx

```diff
@@ -1,33 +1,50 @@
 import * as React from "react";
 import { Formik, FormikHelpers, Form, Field } from "formik";
 import * as Yup from "yup";
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import { DateTimePicker } from "@mui/x-date-pickers/DateTimePicker";
 import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
 import { AdapterDateFns } from "@mui/x-date-pickers/AdapterDateFns";
 import { useRouter } from "next/router";
-import { addEvent } from "@/lib/firebase/firestore";
+import { addEvent, updateEvent } from "@/lib/firebase/firestore";
 import { Timestamp } from "firebase/firestore";
 import { uploadImage } from "@/lib/firebase/storage";
-import { toast } from "react-toastify";
-import { ValidationMessages } from "@/constants/formEnums";
-import { MyFormValues } from "@/interfaces/EventInterface";
+import { showToast } from "@/helpers/toast";
 import { validationSchema } from "@/validation/EventValidation";
+import _ from "lodash";
```

Comment:
@LaithAbusada could you please import what you need here instead of importing the entire `lodash` library? This is to have your generated chunks smaller.